etc/adminhtml/system.xml

```diff
@@ -663,6 +663,60 @@
                         </depends>
                     </field>
                 </group>
+                <group id="omise_cc_googlepay" translate="label" sortOrder="505" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable Google Pay Payment</label>
+                    <comment>Enable customers to pay using credit/debit cards connected to their Google account. </comment>
+                    <field id="active" translate="label comment" type="select" sortOrder="506" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Enable/Disable</label>
+                        <comment>Enable Google Pay payment method.</comment>
+                        <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                        <config_path>payment/omise_cc_googlepay/active</config_path>
+                    </field>
+                    <field id="merchant_id" translate="label" type="text" sortOrder="507" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Google Pay Merchant ID</label>
+                        <comment><![CDATA[The merchant ID will be available after registering with the <a href="https://pay.google.com/business/console">Google Pay Business Console</a>.]]></comment>
```

Comment:
can you note that this is not needed for test mode?

Reply:
Added here: [08e206d](https://github.com/omise/omise-magento/pull/351/commits/08e206dc0a2767c061f61a2d64f65744b56388c2)